software/open-quartermaster.yml

```diff
@@ -0,0 +1,15 @@
+name: "Open QuarterMaster"
+website_url: "https://openquartermaster.com/"
+source_code_url: "https://github.com/Epic-Breakfast-Productions/OpenQuarterMaster"
+description: "A powerful inventory management system, designed to be flexible and scalable."
+licenses:
+  - GPL-3.0
+platforms:
+  - Deb
+  - Docker
```

Comment:
```suggestion
platforms:
  - Deb
```

I can't find Docker-based setup instructions under https://github.com/Epic-Breakfast-Productions/OpenQuarterMaster/tree/main/deployment

Reply:
![image](https://github.com/user-attachments/assets/b84ea591-9a13-44fe-965a-a550dc40cdeb)
I suppose we can remove it as its not there yet, but he said they will add docker compose support, along with k8s soon. I thought it was fine as its on the roadmap.

Reply:
> Everything runs with containers under the hood


I did not see this since it was in the collapsed comments. Disregard my suggestion